cirq/google/common_serialized_gates.py

```diff
@@ -0,0 +1,111 @@
+# Copyright 2019 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Standard way to serialize gates common to initial architectures.
+
+For new devices and gate sets, prefer to use these serialized versions
+in order to standardize the serialization format across multiple
+gate sets.
+"""
+
+from typing import Type, cast, List, TypeVar
+
+from cirq import ops, protocols
+from cirq.google import op_serializer, op_deserializer
+
+Gate = TypeVar('Gate', bound=ops.Gate)
+
+LAMBDA_ZERO = lambda x: 0.0
```

Comment:
There are the same number of characters in `LAMBDA_ZERO` and `lambda _: 0`. Just inline these. Not worth making a constant for.

Reply:
Done.